### ISSUE TITLE
Accesibility

### ISSUE THREAD

**Issue by juan-bastidas** [akiran/react-slick#1980]
Running AXE validation returns one a11y error introduced by react-slick: 

**ARIA hidden element must not contain focusable elements**

Basically stop adding **aria-hidden** to each slide, tabindex=-1 will be the way for non visible slides.

**Comment by garyb1** [comment]
Why not use the Inert attribute? You can use the polyfill from [wicg github](https://github.com/WICG/inert)

**Comment by dushakov92** [comment]
Has anyone found a good work around for this? 
Having links inside slides creates accessibility check failures.
<img width="1791" alt="Screenshot 2023-09-11 at 4 14 24 PM" src="https://github.com/akiran/react-slick/assets/69015238/057a95e7-8dde-4d1e-b264-15719a5401f5">

**Comment by cakidnyc** [comment]
Same problem:

![image](https://github.com/user-attachments/assets/5fd5ffeb-efa4-41ba-a921-88d2ba1bd5a1)
